### PR TITLE
chore: add description in package.json

### PR DESCRIPTION
**Pull request by hyunbinseo**
Following packages show invalid descriptions in the [npm website] with a markdown link syntax.

image

[npm website]: https://www.npmjs.com/

[create-svelte](https://www.npmjs.com/package/create-svelte)
`A CLI for creating new [SvelteKit](https://kit.svelte.dev) projects. Just run...`

[@sveltejs/kit](https://www.npmjs.com/package/@sveltejs/kit)
`This is the [SvelteKit](https://kit.svelte.dev) framework and CLI.`

[@sveltejs/vite-plugin-svelte](https://www.npmjs.com/package/@sveltejs/vite-plugin-svelte)
`The official [Svelte](https://svelte.dev) plugin for [Vite](https://vitejs.dev).`

[@sveltejs/adapter-static](https://www.npmjs.com/package/@sveltejs/adapter-static)
`[Adapter](https://kit.svelte.dev/docs/adapters) for SvelteKit apps that prerenders your entire site as a collection of static files. It's also possible to create an SPA with it by specifying a fallback page which renders an empty shell. If you'd like to p`

[@sveltejs/adapter-node](https://www.npmjs.com/package/@sveltejs/adapter-node)
`[Adapter](https://kit.svelte.dev/docs/adapters) for SvelteKit apps that generates a standalone Node server.`

[@sveltejs/package](https://www.npmjs.com/package/@sveltejs/package)
`Create Svelte packages, build and push them to npm. This is part of the [SvelteKit](https://kit.svelte.dev) framework and CLI.`

[@sveltejs/adapter-cloudflare](https://www.npmjs.com/package/@sveltejs/adapter-cloudflare)
`[Adapter](https://kit.svelte.dev/docs/building-your-app) for building SvelteKit applications on [Cloudflare Pages](https://developers.cloudflare.com/pages/) with [Workers integration](https://developers.cloudflare.com/pages/platform/functions).`

[@sveltejs/adapter-begin](https://www.npmjs.com/package/@sveltejs/adapter-begin)
`Adapter for Svelte apps that creates a [Begin](https://begin.com/) app, using a function for dynamic server rendering.`

---

Since this is not a bug report or a feature request, it is opened as a pull request.

Note that the Svelte package includes the `description` field already.

https://github.com/sveltejs/svelte/blob/e2fe8ab46966a7c3a6702e7e5f71ac316c553288/package.json#L4

---

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### CODE REVIEW

**Review by dummdidumm — APPROVED** [review]
Thank you! I added descriptions for all the packages - will let some other maintainer review those.

File: packages/adapter-cloudflare-workers/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@sveltejs/adapter-cloudflare-workers",
 	"version": "1.1.0",
+	"description": "An adapter for SvelteKit, making it possible to run your app on Cloudflare Workers",
```
Comment (benmccann) [inline packages/adapter-cloudflare-workers/package.json:4]:
```suggestion
	"description": "SvelteKit adapter that creates a Cloudflare Workers site using a function for dynamic server rendering",
```

File: packages/adapter-auto/package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@sveltejs/adapter-auto",
 	"version": "2.0.0",
+	"description": "An zero-config adapter for SvelteKit that automatically picks the right adapter depending on your platform",
```
Comment (benmccann) [inline packages/adapter-auto/package.json:4]:
```suggestion
	"description": "Automatically chooses the SvelteKit adapter for your current environment, if possible.",
```

File: packages/adapter-cloudflare/package.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@sveltejs/adapter-cloudflare",
 	"version": "2.2.0",
+	"description": "An adapter for SvelteKit, making it possible to run your app on Cloudflare",
```
Comment (benmccann) [inline packages/adapter-cloudflare/package.json:4]:
```suggestion
	"description": "Adapter for building SvelteKit applications on Cloudflare Pages with Workers integration",
```

File: packages/adapter-netlify/package.json [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@sveltejs/adapter-netlify",
 	"version": "2.0.6",
+	"description": "An adapter for SvelteKit, making it possible to run your app on Netlify",
```
Comment (benmccann) [inline packages/adapter-netlify/package.json:4]:
```suggestion
	"description": "A SvelteKit adapter that creates a Netlify app",
```

File: packages/adapter-node/package.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@sveltejs/adapter-node",
 	"version": "1.2.2",
+	"description": "An adapter for SvelteKit suitable for Node environments",
```
Comment (benmccann) [inline packages/adapter-node/package.json:4]:
```suggestion
	"description": "Adapter for SvelteKit apps that generates a standalone Node server",
```

File: packages/adapter-static/package.json [hunk 6]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@sveltejs/adapter-static",
 	"version": "2.0.1",
+	"description": "An adapter for SvelteKit suitable for static sites and single page apps",
```
Comment (benmccann) [inline packages/adapter-static/package.json:4]:
```suggestion
	"description": "Adapter for SvelteKit apps that prerenders your entire site as a collection of static files",
```

File: packages/adapter-vercel/package.json [hunk 7]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@sveltejs/adapter-vercel",
 	"version": "2.3.1",
+	"description": "An adapter for SvelteKit, making it possible to run your app on Vercel",
```
Comment (benmccann) [inline packages/adapter-vercel/package.json:4]:
```suggestion
	"description": "A SvelteKit adapter that creates a Vercel app",
```

File: packages/create-svelte/package.json [hunk 8]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "create-svelte",
 	"version": "3.1.2",
+	"description": "A CLI for quickly scaffolding your next Svelte app or library",
```
Comment (benmccann) [inline packages/create-svelte/package.json:4]:
```suggestion
	"description": "A CLI for creating new SvelteKit projects",
```

File: packages/kit/package.json [hunk 9]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@sveltejs/kit",
 	"version": "1.11.0",
+	"description": "SvelteKit is a framework for building web apps, streamlined",
```
Comment (benmccann) [inline packages/kit/package.json:4]:
```suggestion
	"description": "The fastest way to build Svelte apps",
```

File: packages/package/package.json [hunk 10]
```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@sveltejs/package",
 	"version": "2.0.2",
+	"description": "Create Svelte packages, build and publish them to npm",
```
Comment (benmccann) [inline packages/package/package.json:4]:
```suggestion
	"description": "The fastest way to build Svelte packages",
```

**Comment by benmccann** [comment]
How about we use the descriptions from the READMEs? I added suggestions showing what those are (except for AMP which doesn't have a README)

**Pull request merged**